src/test/java/fi/helsinki/cs/tmc/cli/backend/CourseInfoIoTest.java

```diff
@@ -53,4 +57,15 @@ public void loadingFromFileWorks() {
         Assert.assertEquals(this.course.getUsername(), loadedInfo.getUsername());
         Assert.assertEquals(this.course.getCourseName(), loadedInfo.getCourseName());
     }
+
+    @Test
+    public void abortingCourseCreationWorks() {
+        CourseInfoIo.save(this.course, this.courseFile);
+        CourseInfo loadedInfo = CourseInfoIo.load(this.courseFile);
+        Assert.assertNotNull(loadedInfo);
+
+        CourseInfoIo.abortCreatingCourse(new Course("test-course"), Paths.get(tempDir));
+        File courseJson = Paths.get(tempDir).resolve(".tmc.json").toFile();
```

Comment:
Avoid using `java.io.File` in new code and prefer the better alternatives in `java.nio`.

For example, this could be rewritten as:
```java
Path courseJson = Paths.get(tempDir).resolve(".tmc.json");
assertTrue(Files.exist(courseJson));
```

Reply:
:whale: